worlds/civ_6/Container.py

```diff
@@ -43,6 +43,10 @@ def write_contents(self, opened_zipfile: zipfile.ZipFile) -> None:
             opened_zipfile.writestr(filename, yml)
         super().write_contents(opened_zipfile)
 
+def sanitize_value(value: str) -> str:
+  """Removes values that can cause issues in XML"""
+  return value.replace('"', "'").replace('&', 'and')

```

Comment:
Would https://docs.python.org/3/library/xml.sax.utils.html#xml.sax.saxutils.quoteattr work here?

Reply:
I may be misunderstanding how to use it but I ended up with all sorts of goofy outcomes with that 😅 `quoteattr(value, {'"': "'", '&': 'and'})`